supportconfig/plugins/suse_public_cloud

```diff
@@ -161,6 +161,10 @@ function metadata() {
         echo "# /usr/sbin/azuremetadata -t" >> $INFO_FILE
         /usr/sbin/azuremetadata -t >> $INFO_FILE
     fi
+    if [ -e "/usr/bin/azuremetadata" ]; then
+        echo "# /usr/bin/azuremetadata" >> $INFO_FILE
```

Comment:
This shold use `--latest` and we need that option in the azuremetadata implementation, see https://github.com/SUSE-Enceladus/azuremetadata/issues/12